<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialog</class>
 <widget class="QDialog" name="QDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>684</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>684</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Accuracy assessment</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Parameters</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>137</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Ground truth:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>73</width>
          <height>17</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dirección donde se guardara el resultado.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="columnaVerdadera">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>201</width>
          <height>25</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>80</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Map class column:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>CSV que contenga la superficie (Ha) de todas las clases.</string>
        </property>
        <property name="text">
         <string>Land cover map:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>221</width>
          <height>17</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shapefile que contenga muestras de validación, dentro de la tabla de atributos debe de tener dos columnas, la primera con valor real de la clasificación y la segunda el valor del mapa tematico que se decea validar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Reference Data (only shapefile):  </string>
        </property>
       </widget>
       <widget class="QToolButton" name="buscarSuperficie">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>190</y>
          <width>26</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QComboBox" name="columnaValidar">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>110</y>
          <width>201</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="bucarMuestas">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>40</y>
          <width>26</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="textResultado">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>391</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="guardarResultado">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>320</y>
          <width>26</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>333</x>
          <y>9</y>
          <width>26</width>
          <height>93</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="itemSuperficie">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>391</width>
          <height>25</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="itemLayers">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>391</width>
          <height>25</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="columClase">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>391</width>
          <height>25</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>241</width>
          <height>17</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>CSV que contenga la superficie (Ha) de todas las clases.</string>
        </property>
        <property name="text">
         <string>Class column (only shapefile):</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Record</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextEdit" name="reultados"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>10</y>
     <width>211</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="toolTip">
       <string/>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Input parameter&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Reference Data: &lt;/span&gt;vector layer file (shp) with reference data.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Ground truth column:&lt;/span&gt;  Column of type string or numeric with validation data. &lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Map class column:&lt;/span&gt; Column of type string or numeric with map class data to validate. &lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Land cover map: &lt;/span&gt;vector layer file (shp) or raster file (tif) of land cover map to validate.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Class column: &lt;/span&gt;Column of type string or numeric with coverage map classes  (only shapefile).&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Output parameters &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Result:&lt;/span&gt; folder where the results will be save&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="aceptar">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>480</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>671</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
